monitor/rpcmonitor/rpc.go

```diff
@@ -159,15 +159,16 @@ type Server struct {
 
 // HandleEvent Gets called when clients generate events.
 func (s *Server) HandleEvent(eventInfo *EventInfo, result *RPCResponse) error {
-
 	if err := validateEvent(eventInfo); err != nil {
 		return err
 	}
-
 	if eventInfo.HostService && !s.root {
 		return fmt.Errorf("Operation Requires Root Access")
 	}
-
+	strtokens := eventInfo.PUID[:strings.LastIndex(eventInfo.PUID, "/")+1]
+	if _, ferr := os.Stat("/var/run/trireme/linux/" + strtokens); os.IsNotExist(ferr) && eventInfo.EventType != monitor.EventCreate && eventInfo.EventType != monitor.EventStart {
```

Comment:
/var/run/trireme should be a constant 
/var/run/trireme/linux should be another and I dont think we should have this in code anywhere